### PR TITLE
small nrunner refactoring

### PR DESCRIPTION
**Pull request by beraldoleal**
The NextRunner will be a core component of Avocado and we could try to keep the code as clean as possible to facilitate the understanding and readability. This changes only the order of a few methods and moves other ones to the respective classes.

### CODE REVIEW

File: selftests/unit/test_nrunner.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def test_kind_noop(self):
     def test_recipe_noop(self):
         open_mocked = unittest.mock.mock_open(read_data='{"kind": "noop"}')
         with unittest.mock.patch("builtins.open", open_mocked):
-            runnable = nrunner.runnable_from_recipe("fake_path")
+            runnable = nrunner.Runnable.from_recipe("fake_path")
```
Comment (clebergnu) [inline selftests/unit/test_nrunner.py]: This deserves to be part of 696eec293b9b36e3afdfdecd55c7e560e4f077db

File: selftests/unit/test_nrunner.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def test_recipe_exec(self):
                        '"args": ["/etc/profile"], '
                        '"kwargs": {"TERM": "vt3270"}}'))
         with unittest.mock.patch("builtins.open", open_mocked):
-            runnable = nrunner.runnable_from_recipe("fake_path")
+            runnable = nrunner.Runnable.from_recipe("fake_path")
```
Comment (clebergnu) [inline selftests/unit/test_nrunner.py]: Ditto here (this deserves to be part of 696eec293b9b36e3afdfdecd55c7e560e4f077db)

File: selftests/unit/test_nrunner.py [hunk 3]
```diff
@@ -92,14 +92,14 @@ class RunnableFromCommandLineArgs(unittest.TestCase):
 
     def test_noop(self):
         parsed_args = {'kind': 'noop', 'uri': None}
-        runnable = nrunner.runnable_from_args(parsed_args)
+        runnable = nrunner.Runnable.from_args(parsed_args)
```
Comment (clebergnu) [inline selftests/unit/test_nrunner.py]: This (and all other instances of similar changes here) deserves to be part of 69ab640b1d79974971c073518a1197d20f50ee9d.

**Comment by beraldoleal** [comment]
@clebergnu thanks for your comments. Sending a v2.

**Pull request closed without merging**